invenio_users_resources/services/users/config.py

```diff
@@ -86,6 +125,13 @@ class UsersServiceConfig(RecordServiceConfig, ConfiguratorMixin):
         "USERS_RESOURCES_SEARCH_FACETS",
         search_option_cls=UserSearchOptions,
     )
+    # For admin user
+    admin_search = FromConfigSearchOptions(
```

Comment:
```suggestion
    search_all = FromConfigSearchOptions(
```

minor: to match the naming of the resource